scipy/special/_basic.py

```diff
@@ -1446,12 +1446,42 @@ def lqmn(m, n, z):
 def bernoulli(n):
     """Bernoulli numbers B0..Bn (inclusive).
 
+    Parameters
+    ----------
+    n : int
+        Indicated the number of terms in the Bernoulli series to generate
+
+    Returns
+    -------
+    ndarray
+        The Bernoulli numbers [B(0), B(1), ..., B(n)].
```

Comment:
Not sure about the name, but it should say something.
```suggestion
    series : array_like (n,)
        The Bernoulli numbers ``[B(0), B(1), ..., B(n)]``.
```

Reply:
@tupui: according to the [numpydoc specification for the "Returns" section](https://numpydoc.readthedocs.io/en/latest/format.html#returns), a name is not required for the return value.  In fact, the first example there shows just `int` rather than `name : int`.  Also, the return value will always be a numpy array, so it seems correct to give the more specific `ndarray` instead of `array_like`.  `array_like` is appropriate for *inputs*, where the input might be a list, a tuple, a Pandas `Series`, etc.

Reply:
Thanks Warren for the explanations. I was not aware that it was not mandatory to give a name. I will edit.